### PR TITLE
Chore: Client benchmark

### LINKED ISSUE #783: Client Benchmark

### PR DESCRIPTION
**Pull request by dinojohn**
Closes #783

### CODE REVIEW

File: example/src/main/scala/example/ClientBenchmark.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package example
+
+import zhttp.service.{ChannelFactory, Client, EventLoopGroup}
+import zio.{App, ExitCode, URIO, ZIO}
+
+import java.util.concurrent.TimeUnit
+
+object ClientBenchmark extends App {
```
Comment (dinojohn) [inline example/src/main/scala/example/ClientBenchmark.scala:9]: Should we move this file from example ?

File: example/src/main/scala/example/SimpleServer.scala [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package example
+
+import zhttp.http._
+import zhttp.service.Server
+import zio._
+object SimpleServer extends App {
+
+  // Run it like any simple app
+  override def run(args: List[String]): URIO[zio.ZEnv, ExitCode] =
+    Server.start(7777, Http.ok.silent).exitCode
```
Comment (dinojohn) [inline example/src/main/scala/example/SimpleServer.scala]: will add a benchmark with post route.

File: example/src/main/scala/example/ClientBenchmark.scala (hunk 1 above)
Comment (girdharshubham) [reply example/src/main/scala/example/ClientBenchmark.scala:8]: It would make better sense.

**Comment by amitksingh1490** [comment]
Warmup the server before calculating req/sec

Client should make request for 30 secs instead of having a limit on total requests.

File: example/src/main/scala/example/SimpleServer.scala (hunk 2 above)
Comment (dinojohn) [reply example/src/main/scala/example/SimpleServer.scala]: done

**Comment by dinojohn** [comment]
Closing until we are able to shutdown the benchmarking server

**Pull request closed without merging**